chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -301,6 +301,13 @@
   notifies :restart, "service[chef-solr]", :delayed
 end
 
+# Make systemd restart chef services (and their deps) on failure
+utils_systemd_service_restart "rabbitmq-server"
+utils_systemd_service_restart "couchdb"
+utils_systemd_service_restart "chef-solr"
```

Comment:
I would put the service restart call right after the `service` block, as you did with the other services.

Reply:
I didn't do it in this case because I wanted to keep all these lines together (since we actually don't deal with these services in the recipe).

So let's wait for another reviewer and get a 3rd opinion :-)